packages/env/README.md

```diff
@@ -1,37 +1,141 @@
+
+Ensure that Docker is running, then:
+
+```sh
+$ cd /path/to/a/wordpress/plugin
+$ npx wp-env start
+```
+
+The local environment will be available at http://localhost:8000.
+
+## Instructions
+
+### Installation
+
+`wp-env` requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other version of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
+
+After confirming that Docker is installed, you can install `wp-env` globally like so:
 
 ```sh
 $ npm -g i @wordpress/env
+```
 
-$ cd path/to/plugin-or-theme # WordPress install will be in path/to/plugin-or-theme-wordpress.
+You're now ready to use `wp-env`!
 
-$ wp-env --help
+### Starting the environment
 
-wp-env <command>
+First, ensure that Docker is running. You can do this by clicking on the Docker icon in the system tray or menu bar.
 
-Commands:
-  wp-env start [ref]          Starts WordPress for development on port 8888
-                              (​http://localhost:8888​) (override with
-                              WP_ENV_PORT) and tests on port 8889
-                              (​http://localhost:8889​) (override with
-                              WP_ENV_TESTS_PORT). If the current working
-                              directory is a plugin and/or has e2e-tests with
-                              plugins and/or mu-plugins, they will be mounted
-                              appropriately.
-  wp-env stop                 Stops running WordPress for development and tests
-                              and frees the ports.
-  wp-env clean [environment]  Cleans the WordPress databases.
+Then, change to a directory that contains a WordPress plugin or theme:
 
-Options:
-  --help     Show help                                                 [boolean]
-  --version  Show version number                                       [boolean]
+```sh
+$ cd ~/gutenberg
 ```
 
-### `$ wp-env start --help`
+Then, start the local environment:
+
+```sh
+$ wp-env start
+```
+
+Finally, navigate to http://localhost:8000 in your web browser to see WordPress running with the local WordPress plugin or theme running and activated.
+
+### Stopping the environment
+
+To stop the local environment:
+
+```sh
+$ wp-env stop
+```
+
+## Troubleshooting common problems
+
+Many common problems can be fixed by running through the following troubleshooting steps in order:
+
+### 1. Check that `wp-env` is running
+
+First, check that `wp-env` is running. One way to do this is to have Docker print a table with the currently running containers:
+
+```sh
+$ docker ps
+```
+
+In this table, by default, you should see three entries: `wordpress` with port 8888, `wordpress` with port 8889 and `mariadb` with port 3306.
```

Comment:
```suggestion
In this table, by default, you should see three entries: `wordpress` with port 8888, `tests-wordpress` with port 8889 and `mariadb` with port 3306.
```

Reply:
It shows in `docker ps` as `wordpress` but agree this is clearer.